src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.view;
+
+import java.util.List;
+
+public class OutputView {
+    private static final String PROCESS_PRINT_START_SENTENCE = "실행 결과";
+    private static final String WINNER_PRINT_START_SENTENCE = "최종 우승자 : ";
+    private static final String PROCESS_BAR = "-";
+    private static final String COLON = " : ";
+    private static final String COMMA = ", ";
+
+
+    public static void processPrintStartMessage() {
+        printEmptyLine();
+        System.out.println(PROCESS_PRINT_START_SENTENCE);
+    }
+
+
+    public static void printCarName(String name) {
+        System.out.print(name + COLON);
+    }
+
+    public static void printCarLocation(int location) {
+        System.out.println(PROCESS_BAR.repeat(location));
+    }
+
+    public static void printEmptyLine() {
+        System.out.println();
+    }
+
+
+    public static void printWinner(List<String> winners) {
+        StringBuilder message = new StringBuilder();
+
+        message.append(WINNER_PRINT_START_SENTENCE);
+        winners.stream().forEach(name -> message.append(name).append(COMMA));
+        message.delete(message.length() - 2, message.length() - 1);
```

Comment:
```suggestion
        String.join(COMMA, winners);
```

이렇게도 줄여볼 수 있지 않을까요?? 😄

Reply:
헉... 이런 함수가 있었군요! 다음부터는 자바에서 제공하는 함수가 있는지 먼저 확인을 해봐야겠어요 :)